charts/hivemq-platform-operator/values.yaml

```diff
@@ -26,6 +26,10 @@ http:
   port: 8080
 https:
   port: 8443
+  # Configures if certificate validation and hostname verification should be skipped for all managed custom resources.
+  # These settings are used for all operator related HTTP clients, e.g. for downloading custom extensions and extension customizations.
```

Comment:
Hmm, I think it would depend on if we refer to the HiveMQ Platform Operator as product name, or just operator as a general operator. I'll align it with the uppercase variant for now.